src/app/components/ThemeProvider/chameleonLogos/polska.tsx

```diff
@@ -2,13 +2,25 @@ import React from 'react';
-      <path d="M1235.917 229.715c0 4.758-1.209 8.828-3.624 12.21-2.416 3.382-5.873 5.983-10.369 7.804-4.5 1.822-9.868 2.732-16.112 2.732h-31.332v-79.723h29.547c8.77 0 15.555 1.728 20.35 5.183 4.794 3.457 7.192 8.458 7.192 14.999 0 3.94-.949 7.341-2.844 10.2-1.896 2.864-4.665 5.112-8.307 6.747 4.98 1.488 8.809 3.942 11.485 7.36 2.676 3.42 4.014 7.582 4.014 12.488zm-47.723-46.163V206.3h14.495c4.832 0 8.548-1.04 11.15-3.121 2.602-2.08 3.903-4.981 3.903-8.698 0-7.285-4.98-10.928-14.941-10.928h-14.607zm0 33.23v24.863h16.39c5.501 0 9.758-1.059 12.768-3.178 3.01-2.117 4.516-5.147 4.516-9.086 0-8.4-5.686-12.6-17.06-12.6h-16.614zM1261.786 221.91v30.551h-13.938v-79.723h27.43c10.33 0 18.247 2.136 23.75 6.41 5.5 4.275 8.25 10.354 8.25 18.23 0 5.28-1.339 9.796-4.014 13.548-2.676 3.756-6.505 6.562-11.485 8.42a877.288 877.288 0 0110.37 16.277 725.842 725.842 0 0110.146 16.838h-16.278a616.243 616.243 0 00-8.865-15.387 1856.792 1856.792 0 00-9.2-15.275c-.89.074-1.821.111-2.786.111h-13.38zm0-37.91v26.648h12.6c6.168 0 10.815-1.06 13.937-3.177 3.122-2.12 4.683-5.483 4.683-10.092 0-4.608-1.56-7.99-4.683-10.147-3.122-2.154-7.732-3.232-13.826-3.232h-12.711zM1382.096 252.461a757.49 757.49 0 00-6.914-19.402h-34.9a424.842 424.842 0 00-3.456 9.646c-1.116 3.233-2.23 6.485-3.345 9.756h-14.942a1552.208 1552.208 0 0115.666-40.922 1281.066 1281.066 0 0116.224-38.801h15.052a2022.784 2022.784 0 0116.447 39.916 2148.038 2148.038 0 0115.443 39.807h-15.275zm-37.466-31.22h26.204a2376.855 2376.855 0 00-6.467-17.396 1181.21 1181.21 0 00-6.58-17.058 780.239 780.239 0 00-6.745 17.115 956.214 956.214 0 00-6.412 17.338zM1429.371 254.133c-4.311 0-8.585-.484-12.822-1.45-4.236-.964-8.214-2.452-11.93-4.46v-12.934c7.655 4.758 15.98 7.137 24.976 7.137 5.575 0 9.867-1.041 12.878-3.123 3.01-2.08 4.516-4.942 4.516-8.584 0-2.08-.446-3.809-1.338-5.186-.892-1.375-2.1-2.525-3.624-3.457-1.525-.928-3.272-1.69-5.24-2.285a114.32 114.32 0 00-6.189-1.672c-3.94-.967-7.525-2.064-10.76-3.289-3.233-1.227-6.02-2.695-8.362-4.404-2.342-1.711-4.145-3.829-5.408-6.356-1.265-2.527-1.895-5.613-1.895-9.256 0-4.906 1.226-9.123 3.679-12.654 2.453-3.532 5.946-6.262 10.481-8.196 4.533-1.933 9.924-2.9 16.168-2.9 4.087 0 8.14.484 12.154 1.451 4.013.967 7.582 2.268 10.704 3.902v12.6c-2.975-1.857-6.45-3.363-10.426-4.515-3.977-1.153-8.195-1.729-12.655-1.729-5.056 0-8.996.947-11.819 2.842-2.826 1.896-4.237 4.592-4.237 8.084 0 2.53.669 4.516 2.007 5.967 1.338 1.449 3.233 2.601 5.686 3.455 2.453.855 5.314 1.691 8.586 2.51a121.7 121.7 0 0110.203 3.01c3.233 1.115 6.093 2.546 8.585 4.292a18.627 18.627 0 015.854 6.635c1.411 2.676 2.118 6.022 2.118 10.036 0 5.129-1.264 9.515-3.79 13.156-2.529 3.643-6.19 6.45-10.983 8.418-4.795 1.969-10.501 2.955-17.117 2.955zM1473.416 172.738h14.16v79.723h-14.16v-79.723zM1500.51 172.738h14.16v67.458h39.249v12.265h-53.41v-79.723z" />
-    </>
+    <g transform="translate(30,0)">
```

Comment:
Is this okay ? Me and Pete have gone back and forth and tried 3 different svg files, each one was getting clipped in some way by the viewbox - this fixes the clipping, but i'm very aware no other svg file does this at all and all seemed to just work without any editing from the devs being needed? It also seems like a massively long svg compared to every other one (other than russian).  I don't know enough about svgs to understand why this might have been happening, or how to update the paths instead of doing a straight transform. Also weirdly, i tried just setting the viewbox manually here (like in the news svg) to the correct one instead of the transform , but this didn't work despite the values being functionally the same as what this results in lol.

Reply:
I reckon this is fine, file size is very similar so it shouldn't be a problem